src/pages/Home/Home.scss

```diff
@@ -0,0 +1,236 @@
+  width: 100%;
+  max-width: 224px;
+  height: 20px;
+  display: flex;
+  box-shadow: 4px 4px 8px rgba(0, 0, 0, .25);
+  
+  @include desktop{
+    max-width: 240px;
+  }
+
+  & input {
+    @include heading-quaternary;
+    height: 100%;
+    width: 100%;
+    border: 0;
+    background-color: $color-field;
+    outline: 0;
+    padding: 2% 4%;
+
+    &:focus{
+      border-bottom: 1px solid $color-emph-primary;
+      box-shadow: 0 -2px 2px inset rgba(0, 0, 0, .25);
+    }
+  }
+
+  & button {
+    height: 20px;
+    min-width: 20px;
+    background-color: $color-emph-primary;
+    border: 0;
+    color: $color-field;
+
+    &:hover{
+      cursor: pointer;
+      box-shadow: 4px 4px 8px rgba(0, 0, 0, .5);
+    }
+  }
+}
+
+.about-intro-wrapper{
+  display: flex;
+  width: 100%;
+  position: relative;
+}
+
+.about-intro {
+  padding: 5% 15%;
+  height: 30vh;
+  @include responsiveMargin(l,top);
+  flex: 1;
+
+  @include tablet {
+    padding-right: 5%;
+  }
+
+  @include desktop {
+    padding-right: 10%;
+  }
+
+  &__title{
+    @include heading-primary;
+    color: $text-emphasis;
+  }
+
+  &__summary{
+    @include responsiveMargin(m,top);
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__msg {
+    @include heading-quaternary;
+    color: $text-offset;
+  }
+
+  &__more {
+    @include heading-secondary;
+    text-decoration: none;
+    color: $text-emphasis;
+    align-self: flex-end;
+    height: 24px;
+  }
+}
+
+.about-intro-image{
+  display: none;
+  background-image: url('../../assets/images/business-man.jpeg');
+  background-size: cover;
+  background-position: 35%;
+  background-repeat: no-repeat;
+  clip-path: url(#svgPathAbout);
+  flex: 1.2;
+
+  @include tablet{
+    display: flex;
+  }
+}
+
+.svg-icon__chevron{
+  height: 100%;
+  margin-left: $margin-xs;
+  position: relative;
+  top: 25%;
+}
+
+.faq-intro-wrapper{
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  @include tablet{
+    flex-direction: row;
+    align-items: flex-end;
+  }
+}
+
+.faq-intro{
+  padding: 0 15%;
+  height: 30vmax;
+  flex: 1.1;
+
+  @include tablet {
+    padding-left: 5%;
+    max-height: 30vh;
+  }
+
+  @include desktop {
+    padding-left: 10%;
+    max-height: 50vh;
+  }
+
+  &__title{
+    @include heading-primary;
+    color: $text-emphasis;
+  }
+
+  &__summary{
+    @include responsiveMargin(m,top);
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    max-width: 310px;
+  }
+
+  &__msg {
+    @include heading-quaternary;
+    color: $text-offset;
+  }
+
+  &__link-wrapper{
+    @include responsiveMargin(s,top);
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+}
+
+.faq-intro-image{
+  display: none;
+  background-image: url('../../assets/images/books.jpeg');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  clip-path: url(#svgPathFaq);
+  flex: 1.25;
```

Comment:
Where we are only passing one variable, is it best to avoid using shorthand? 
Concerned that this is not explicit.

Reply:
As discussed, agreed.

Reply:
Just realized a misunderstanding on my part. I had thought of flex-shrink as the second value but the second value is flex-shrink _or_ flex-basis.  flex: 1.25 is the equivalent of:
```
flex-grow: 1.25;
flex-basis: 0;
```
As such the shorthand is a bit more concise here but we could also do:
```
flex: 1.25 0;
```